tests/operator/operator.go

```diff
@@ -764,12 +764,15 @@ var _ = Describe("[Serial][sig-operator]Operator", Serial, decorators.SigOperato
 			return vmis
 		}
 
-		startAllVMIs = func(vmis []*v1.VirtualMachineInstance) {
-			for _, vmi := range vmis {
-				vmi, err := virtClient.VirtualMachineInstance(testsuite.GetTestNamespace(vmi)).Create(context.Background(), vmi, metav1.CreateOptions{})
+		startAllVMIs = func(vmis []*v1.VirtualMachineInstance) []*v1.VirtualMachineInstance {
+			newVMIs := make([]*v1.VirtualMachineInstance, len(vmis))
```

Comment:
Not agree this time. append should be use if you don't know the final length, because append is pricey. Here we do know the final length and there is no reason to use append.

Reply:
And yes, I know it's "only" a test.